os/kernel/task/task_dumptasks.c

```diff
@@ -106,8 +109,10 @@ void task_show_tcbinfo(struct tcb_s *tcb)
 	lldbg("Sigdeliver  : %p\n", tcb->xcp.sigdeliver);
 #endif
 #ifdef CONFIG_LIB_SYSCALL
-	lldbg("Nsyscalls   : %u\n", tcb->xcp.nsyscalls);
-	lldbg("Syscall     : %p\n", tcb->xcp.syscall);
+	lldbg("Nsyscalls   : %u\n", nsyscall);
+	for (int i = 0; i < nsyscall; i++) {
+		lldbg("Syscall %d   : %p\n", i+1, tcb->xcp.syscall[i].sysreturn);
```

Comment:
1. i + 1
2. Could you let me know why you print the value with `+1`?

Reply:
Oh.. ok. It's matched with nsyscall.
I found some assert code prints different index number. I don't remember what it is but current assert prints now. Could you check and use the same index range?
I mean in assert code, same buffer is printed twice.
First, it prints index `0~1`. Second, it prints index `1~2`.

Reply:
okay.. I have changed the indexing based upon the indexing present in the base code. That is 0-based indexing for syscall.